package.json

```diff
@@ -2,7 +2,7 @@
   "name": "jsroot",
   "version": "7.5.0",
   "engines": {
-    "node": ">=0.18.0"
```

Comment:
This is fix because `0.18.0` is actually Node's major version 0, and minor version 18 😅